### PR TITLE
VY: update virtual_y_editor permissions

### PR DESCRIPTION
**Pull request by jediandrey**

### CODE REVIEW

**Comment by jediandrey** [comment]
retest

**Review by hamrant — APPROVED** [review]

File: openy_gated_content.install [hunk 1]
```diff
@@ -158,3 +158,16 @@ function openy_gated_content_update_8007() {
     }
   }
 }
+
+/**
+ * Update virtual_ymca_editor user role.
+ */
+function openy_gated_content_update_8008() {
+  $config_dir = drupal_get_path('module', 'openy_gc_storage');
```
Comment (hamrant) [inline openy_gated_content.install:166]: Wrong module name, should be `openy_gated_content`

**Review by hamrant — DISMISSED** [review]

File: openy_gated_content.install (hunk 1 above)
Comment (anpolimus) [inline openy_gated_content.install:166]:
```suggestion
  $config_dir = drupal_get_path('module', 'openy_gated_content');
```

Comment (anpolimus) [reply openy_gated_content.install]: I've did this fix

**Review by anpolimus — APPROVED** [review]

**Comment by anpolimus** [comment]
image
Tested at build - everything works as expected.

**Review by hamrant — APPROVED** [review]

**Pull request merged**